pkg/shell/terraform.go

```diff
@@ -181,19 +182,17 @@ func ExportOutputs(tf *tfexec.Terraform, metadataFile string, artifactsDir strin
 // ImportInputs will search artifactsDir for files produced by ExportOutputs and
 // combine/filter them for the input values needed by the group in the Terraform
 // working directory
-func ImportInputs(deploymentGroupDir string, metadataFile string, artifactsDir string) error {
-	deploymentRoot := path.Clean(path.Join(deploymentGroupDir, ".."))
-	thisGroup := path.Base(deploymentGroupDir)
+func ImportInputs(deploymentGroupDir string, artifactsDir string, expandedBlueprintFile string) error {
+	deploymentRoot := filepath.Clean(filepath.Join(deploymentGroupDir, ".."))
+	thisGroup := filepath.Base(deploymentGroupDir)
 
-	outputNamesByGroup, err := getIntergroupOutputNamesByGroup(thisGroup, metadataFile)
+	outputNamesByGroup, err := getIntergroupOutputNamesByGroup(thisGroup, expandedBlueprintFile)
```

Comment:
nit. `getIntergroupOutputNamesByGroup` and `GetDeploymentKinds` both export blueprint, it seems to be reasonable to export it once. Not for performance reason but, for reducing scope of functions.

Reply:
I had same thought. Would it better to have two funcs more like:

```
func (bp Blueprint) GetDeploymentKinds()

func (bp Blueprint) GetIntergroupOutputNamesByGroup(inputGroup string)
```